blockchain/miner.py

```diff
@@ -24,6 +24,8 @@ def proof_of_work(last_proof):
     print("Searching for next proof")
     proof = 0
     #  TODO: Your code here
+    while valid_proof(last_proof, proof):
```

Comment:
`valid_proof(last_hash, proof)` takes in last_hash and not last_proof. You are passing in a number but valid_proof requires a hash